packages/dd-trace/src/telemetry/index.js

```diff
@@ -7,6 +7,7 @@ const dependencies = require('./dependencies')
 const { sendData } = require('./send-data')
 
 const { manager: metricsManager } = require('./metrics')
+const logs = require('./log')
```

Comment:
requiring `log` to varname `logs`, sounds weird.

Reply:
I had to rename the module as `log` because `logs` are ignored in .gitignore but I'm about to rename it again as `logs` and include `telemetry/logs` in .gitignore. wdyt?

Reply:
renamed :open_mouth: